### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by dcr-stripe**
Codegen for openapi 737a5b2.
r? @richardm-stripe
cc @stripe/api-libraries

## Changelog
* Add support for `payment_settings` on `Subscription` and `SubscriptionParams`.

### CODE REVIEW

File: sub.go [hunk 1]
```diff
@@ -40,6 +40,47 @@ const (
...
 
+// Preferred language of the Bancontact authorization page that the customer is redirected to.
+type SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage string
+
+// List of values that SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage can take
+const (
+	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageDe SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "de"
+	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageEn SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "en"
+	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageFr SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "fr"
+	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageNl SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "nl"
```
Comment (richardm-stripe) [inline sub.go:51]: I think these are acronyms and need to be properly gophercased
```suggestion
const (
	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageDE SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "de"
	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageEN SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "en"
	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageFR SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "fr"
	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageNL SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "nl"
```

Comment (richardm-stripe) [reply sub.go:51]: Although elsewhere, it looks like we've kind of punted on this problem and just have "preferred_language" as a string. I think that probably makes sense here, too.

File: sub.go [hunk 2]
```diff
@@ -40,6 +40,47 @@ const (
...
+)
+
+// The list of payment method types to provide to every invoice created by the subscription. If not set, Stripe attempts to automatically determine the types to use by looking at the invoice's default payment method, the subscription's default payment method, the customer's default payment method, and your [invoice template settings](https://dashboard.stripe.com/settings/billing/invoice).
+type SubscriptionPaymentSettingsPaymentMethodType string
+
+// List of values that SubscriptionPaymentSettingsPaymentMethodType can take
+const (
+	SubscriptionPaymentSettingsPaymentMethodTypeAchCreditTransfer  SubscriptionPaymentSettingsPaymentMethodType = "ach_credit_transfer"
+	SubscriptionPaymentSettingsPaymentMethodTypeAchDebit           SubscriptionPaymentSettingsPaymentMethodType = "ach_debit"
+	SubscriptionPaymentSettingsPaymentMethodTypeAuBECSDebit        SubscriptionPaymentSettingsPaymentMethodType = "au_becs_debit"
```
Comment (richardm-stripe) [inline sub.go:70]:
```suggestion
	SubscriptionPaymentSettingsPaymentMethodTypeAUBECSDebit        SubscriptionPaymentSettingsPaymentMethodType = "au_becs_debit"
```

**Review by richardm-stripe — CHANGES_REQUESTED** [review]
Two minor acronym-related things:
Let's override `BancontactPreferredLanguage` to be a string, and fix `AuBECS

**Comment by dcr-stripe** [comment]
Thanks @richardm-stripe . Fixed! PTAL?

**Review by richardm-stripe — APPROVED** [review]

**Pull request merged**